WordPress/src/main/java/org/wordpress/android/ui/domains/usecases/FetchAllDomainsUseCase.kt

```diff
@@ -0,0 +1,30 @@
+package org.wordpress.android.ui.domains.usecases
+
+import org.wordpress.android.fluxc.network.rest.wpcom.site.AllDomainsDomain
+import org.wordpress.android.fluxc.store.SiteStore
+import javax.inject.Inject
+
+class FetchAllDomainsUseCase  @Inject constructor(
+    private val siteStore: SiteStore,
+) {
+    suspend fun execute(): AllDomains {
+        val result = siteStore.fetchAllDomains()
+        return when {
+            result.isError -> AllDomains.Error
+            result.domains.isNullOrEmpty() -> return AllDomains.Empty
```

Comment:
`return` is redundant here

Reply:
Thank you for checking this PR and your feedback @justtwago 🙇 
Great catch! I opened a small PR to fix this and cross it off the list ;)
https://github.com/wordpress-mobile/WordPress-Android/pull/19401